python-phoenixdb/phoenixdb/tests/dbapi20.py

```diff
@@ -832,3 +832,18 @@ def test_ROWID(self):
         self.assertTrue(hasattr(self.driver,'ROWID'),
             'module.ROWID must be defined.'
             )
+
+    # https://issues.apache.org/jira/browse/PHOENIX-6917
```

Comment:
This file was adopted from an external test suite, and by convention tests here use the global 'barfly' and 'booze' tables.

Please either keep that convention, or move this test to test_db.py, and use the DatabaseTestCase plumbing for opening connections and cleaning up tables.

Doing either ensures that there are no tables left dangling after the test suite has run.

Reply:
Otherwise re-running the tests on the same instance will fail.